bmc-acf/acf_manager.hpp

```diff
@@ -0,0 +1,72 @@
+#pragma once
+
+#include <sdbusplus/bus.hpp>
+#include <sdbusplus/server/object.hpp>
+#include <sdeventplus/source/event.hpp>
+#include <xyz/openbmc_project/Certs/ACF/server.hpp>
+typedef std::tuple<std::vector<uint8_t>, bool, std::string> acf_info;
+
+namespace acf
+{
+namespace cert
+{
+
+class ACFCertMgr;
+
+using CreateIface = sdbusplus::server::object::object<
+    sdbusplus::xyz::openbmc_project::Certs::server::ACF>;
```

Comment:
This nomenclature will likely need to be updated to what Patrick has done upstream, like "sdbusplus::server::xyz::openbmc_project::certs::acf"

Reply:
Yeah, my snapshot for 1110-ghe is about 2-3 weeks back so I think what's best is to get what we have merged and then start tacking periodic rebases (that will require updates like you noted here)